Removed more ecto crap, added tailwindcss & change name

emmahyde opened the pull request.

emmahyde committed:
working on getting the phoenix page to actually be anything useful

Shaka-n reviewed (dismissed):
I understand what's happening here. No need for Ecto when we're not using a database. Tailwind is our CSS framework, since we are definitely not styling this by hand.

emmahyde committed:
groupstrip is now batchstrip

emmahyde committed:
production-ready dockerfile

Testing as a part of the build process

fix tests

break tests on purpose

fix tests on purpose
